src/components/PasswordInput.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { TextField } from "@mui/material";
+import { FieldErrors, UseFormRegister } from "react-hook-form";
+import { LogInInputs } from "../types/LoginInput";
+
+interface PasswordInputProps {
+  register: UseFormRegister<LogInInputs>;
+  errors: FieldErrors<LogInInputs>;
+}
+
+const PasswordInput: React.FC<PasswordInputProps> = ({ register, errors }) => {
+  return (
+    // 8 char min, allows pasting, no char type limit, no composition rules
```

Comment:
Good requirements here, especially that you explicitly allow pasting the password :+1: